tests/zfs-tests/tests/functional/cli_root/zfs_mount/zfs_mount_013_pos.ksh

```diff
@@ -57,6 +57,17 @@ log_must eval ZFS_MOUNT_HELPER=1 zfs $mountcmd -v $TESTPOOL
 log_must mounted $TESTPOOL
 force_unmount $TESTPOOL
 
+log_note "Verify CWD prefix filter <dataset> <path>"
+cd / || log_fail
+zfs set mountpoint=legacy $TESTPOOL || log_fail
+mkdir -p $mntpoint || log_fail
```

Comment:
nit: actually all of these `.. || log_fail` lines could be converted to use the `log_must` debug function.  Basically any line starting with `log_must ...` will execute the command and if it returns a non-zero error code log the failed command, call the cleanup function, and exit the script.

i.e.: log_must mkdir -p $mntpoint

Reply:
They could, but it isn't the same thing. `log_must` logs both success and failure, `.. || log_fail` only logs failure. Do you really want a log entry for a successfully executed `cd /`?

Reply:
That's true.  Personally, I'd say yes since changing the directory is a critical part of the test in this case.  Plus it's more consistent with all the other test cases.  But I don't feel strongly about it.